src/test/java/teammates/test/cases/datatransfer/SessionAttributesTest.java

```diff
@@ -1,124 +0,0 @@
-package teammates.test.cases.datatransfer;
-
-import java.time.Instant;
-import java.util.ArrayList;
-
-import org.testng.annotations.Test;
-
-import teammates.common.datatransfer.attributes.SessionAttributes;
-import teammates.common.util.TimeHelper;
-import teammates.test.cases.BaseTestCase;
-
-/**
- * SUT: {@link SessionAttributes}.
- */
-public class SessionAttributesTest extends BaseTestCase {
-
-    @Test
-    public void testSort() {
```

Comment:
Should the test for `DESCENDING_ORDER` sorter be migrated to `FeedbackSessionAttributesTest.java`?